<?xml version="2.0"?>


<launch>
    <arg name="gazebo" default="false" />

    <group unless="$(arg gazebo)">
        <node name="robot_movement_generation" type="baseline" pkg="human_robot_collaborative_learning" output="screen">
            <rosparam file="$(find human_robot_collaborative_learning)/config/baseline.yaml" command="load"/>
        </node>
    </group>

    <group if="$(arg gazebo)">
        <node name="robot_movement_generation" type="baseline" pkg="human_robot_collaborative_learning" output="screen">
            <rosparam file="$(find human_robot_collaborative_learning)/config/baseline.yaml" command="load"/>
            <remap from="/ur3_cartesian_velocity_controller/command_cart_vel" to="/ur3_cartesian_velocity_controller_sim/command_cart_vel" />
            <remap from="/ur3_cartesian_velocity_controller/ee_state" to="/ur3_cartesian_velocity_controller_sim/ee_state" />
        </node>
    </group>

    <node name="score_visualization" type="score_visualization.py" pkg="human_robot_collaborative_learning" output="screen">
    </node>
</launch>

